src/app/components/home/home.component.scss

```diff
@@ -0,0 +1,132 @@
+html {
+  height: 100%;
+}
+
+body {
+  padding-top: 5rem;
+  height: 100%;
+}
+
+.content {
+  padding-top: 64px;
+  height: 100%;
+  color: white;
+  font-family: spotify-circular;
+}
+
+#header-div {
+  padding-top: 40px;
+}
+
+#myCarousel {
+  padding-bottom: 10px;
+  margin-bottom: 40px;
+}
+
+.starter-template {
+  padding: 2rem 1.5rem;
+  text-align: center;
+}
+
+/* GLOBAL STYLES
```

Comment:
I'm assuming that these are comments from the stylesheet you borrowed. We can leave these in.